src/lib/url/filter.js

```diff
@@ -77,7 +77,7 @@ function expandItem(filterIndex, filterItem) {
 }
 
 export const expandFilter = unexpandedFilter => {
-  let filter = Object.assign({}, unexpandedFilter);
+  let filter = JSON.parse(JSON.stringify(unexpandedFilter));
```

Comment:
Maybe benefit from a "deep copy" method in utilities, there's several places we were using the `Object.assign` trick assuming it included nested objects.